README.md

```diff
@@ -48,35 +49,53 @@ $ ./build-agent.sh
 
 (In the future, Reflect may release an official container image publicly.)
 
+Alternatively, you can run the agent directly on your host machine running Mac OS X.
+In this mode, the agent runs as a collection of bash scripts,
+but requires several utility program dependencies to be installed.
+
+You can check whether the dependencies are installed using:
+
+```
+$ ./local/check-dependencies.sh
+```
+
+Most dependencies are easily installed using a package manager, such as `brew`.
+However, there are installation scripts for some dependencies, such as:
+
+```
+$ ./local/install-dependency-3proxy.sh
+```
+
 ## Running the Agent
 
 To run the agent, you'll need your Reflect account API key,
 which can be found on the __Settings__ page in the Reflect web UI.
 
 Additionally, you can optionally specify the public UDP port that the agent
 will bind to in order to listen for connections from Reflect cloud browsers.
-If you're running the agent on a host behind a NAT,
-you can specify the `-n` flag to improve the reliability of the connection.
 
 Then, run the agent using the following command usage options:
 
 ```
-$ ./run-agent.sh -k <reflect_api_key> [-p <public_port] [-n]
+$ ./run-agent.sh [--local] -k <reflect_api_key> [-p <public_port>]
 	Runs the Reflect Agent and connects to the specified Reflect account
 
+	--local
+		Runs the agent without Docker isolation directly on the host machine.
+		This requires installing several utility program dependencies.
+		See the `local/check-dependency.sh` and `local/install...` scripts.
+		NOTE: this mode requires 'sudo' since it modifies network interfaces.
```

Comment:
I think the comment about `sudo` makes sense after reading it again. For some reason when I ran it the first time I thought it meant that one of the subcommands would use sudo and prompt me to login (which did happen), not that i needed to use `sudo` to invoke `./run-agent.sh`.

Reply:
Cool - you may be thinking of the prompt in the installation instructions for the proxy:
```
echo "Installation to ($Destination) requires sudo, and is assumed to be on the PATH."
```
My thinking was that it makes sense to request the sudo access only for that lone command (moving the binary) rather than requiring the script to run with it.